kattis2canvas.py

```diff
@@ -256,16 +256,38 @@ def course2canvas(offering, canvas_course, dryrun, force, add_to_module):
             break
 
     if not kattis_group:
-        error(f"no kattis assignment group in {canvas_course}")
-        exit(4)
+        # create assignment group if not present on canvas
```

Comment:
it's simpler to use https://canvasapi.readthedocs.io/en/stable/course-ref.html#canvasapi.course.Course.create_assignment_group :)

also, can you put this behind a --create-group flag? and besure to check the dryrun flag

Reply:
Sure, Ben, I can do the API changes.

But the assignment group creation is implicit, it gets created if no group name "kattis" exists.

Shall I modify the logic of not having only "kattis" named assignment group, but a customized group name? Since if we only want one assignment group for kattis assignments then the current logic can be continued by hardcoding the group name in code. Please suggest.